<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuracion</class>
 <widget class="QMainWindow" name="Configuracion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="contenedor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>841</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="formularioTerminar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>371</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 127);
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="tamMem">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>145</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TamanoMemoria:</string>
      </property>
     </widget>
     <widget class="QLabel" name="tamSo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TamanoSistOp:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="selectAlg">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Seleccione algoritmo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BestFit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FirstFit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Worstfit</string>
       </property>
      </item>
     </widget>
     <widget class="QRadioButton" name="radioButtonFija">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>390</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fija</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonVariable">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>390</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Variable</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleccione tipo de Particion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleccione Algoritmo</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxTamMemo">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>160</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxTamSo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAceptarConf">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>510</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(221, 221, 221);</string>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonReiniciarConf">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>510</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(221, 221, 221);</string>
      </property>
      <property name="text">
       <string>Reiniciar</string>
      </property>
     </widget>
     <widget class="QLabel" name="tamMem_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>NombreConf:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineNombre">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Memoria</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="tamMem_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>NumeroParticiones:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxNumPart">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>440</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>841</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Configuracion de Arranque&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="formularioTerminar_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>371</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 127);
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="tamMem_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>NombreConf:</string>
      </property>
     </widget>
     <widget class="QLabel" name="tamMem_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TipoParticion:</string>
      </property>
     </widget>
     <widget class="QLabel" name="tamMem_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>MemoriaRestante:</string>
      </property>
     </widget>
     <widget class="QLabel" name="tamMem_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PuntoInicion:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxNumPart_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="tamMem_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PuntoFin:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxNumPart_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonReiniciarConf_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>510</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(221, 221, 221);</string>
      </property>
      <property name="text">
       <string>Reiniciar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAceptarConf_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(221, 221, 221);</string>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonReiniciarConf_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>510</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(221, 221, 221);</string>
      </property>
      <property name="text">
       <string>Terminar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Particion</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
